src/gui/qgssearchquerybuilder.cpp

```diff
@@ -63,7 +63,8 @@ QgsSearchQueryBuilder::QgsSearchQueryBuilder( QgsVectorLayer *layer,
   setupListViews();
   connect( buttonBox, &QDialogButtonBox::helpRequested, this, &QgsSearchQueryBuilder::showHelp );
 
-  setWindowTitle( tr( "Search Query Builder" ) );
```

Comment:
I have no idea where this dialog is used. I couldn't find the way to the caller...

Reply:
It's deprecated and marked for removal in 4.0 -- so only possibly in use by a plugin, not by QGIS core

Reply:
Yes thanks, i'd finally found the info later, in the .h file.
Then should I turn the title into "Search Provider Filter - xxx" (assuming this is meaningful and reflects what the dialog is meant to do)? `Search filter on provider`, `Search (strings) on provider`? Or leave as is?
Sorry for the noob question... I have no real idea what the dialog does.